core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -551,6 +551,14 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.SERVER)
           .build();
+  public static final PropertyKey WEB_THREAD_DUMP_TO_LOG =
```

Comment:
How many lines of log will this print?

Reply:
All threads will be print here without limit.

Reply:
I've seen many deployments have 1000 or even 2000 threads. This means you are dumping at least 1000*50 lines at once. I don't think I want 50K lines of thread dumps while I'm checking the logs. What is the use case here? Do you know @maobaolong ?

Reply:
@jiacheliu3 How about set this default as false? It would not be a good idea if put 50k line into logger as default.

Reply:
I don't know why require printing thread information to the log and why the user need to get it from the log when they can already get it from webui.

Reply:
@LuQQiu Do you think a separate logger and log destination will be preferable? Or at least there should be a rate limiter on this request. If I blindly refresh on this page, i can DDoS  the server in less than 10 requests.

Reply:
@tieujason330 how much overhead would there be to add a button in the webUI to explicitly ask the process to dump it?

Reply:
adding a button would be straightfoward:
- add button on stacks page
- create redux action + saga to send POST request to server
- server needs new POST handler that'll tell process to dump

Reply:
I think a cache could be added to prevent performance problems caused by multiple refreshes. @jiacheliu3

Reply:
Can I make this a standalone assignment, which kind of defeats the purpose of this PR in the first place?